### ISSUE TITLE
multiple file upload

### ISSUE THREAD

**Issue by benib** [bolt/boltforms#48]
Thanks for the nice extension. It would be useful if multiple file uploads would be handled.
If I do this:

``` yml
documents:
  type: file
  options:
  required: false
  label: Documents
  attr:
    multiple: true
```

I can upload multiple files as the attribute gets added to the input element, but the extension only handles one (the last one selected?)

**Comment by jadwigo** [comment]
Small nitpick - the syntax should be the same as symfony forms expects:

```yml
    image:
        type: file
        options:
            multiple: true
            required: false
            label: Upload your images
            attr:
                class: my-custom-upload
```

**Comment by GawainLynch** [comment]
This is implemented in 4.0.0-beta1 :+1:
